configs/default/HBRO-KAKUTEH7mini.config

```diff
@@ -0,0 +1,145 @@
+# Betaflight / STM32H743 (SH74) 4.2.11 Nov  9 2021 / 20:29:57 (948ba6339) MSP API: 1.43
+
+board_name KAKUTEH7MINI
```

Comment:
board name does not match with the file name. Thats why build is crashing, check the build logs plz